packages/realm-server/scripts/convert-to-sqlite.ts

```diff
@@ -0,0 +1,202 @@
+/* eslint-env node */
+import { readFileSync, readdirSync, writeFileSync } from 'fs-extra';
+import { resolve, join } from 'path';
+import {
+  parse,
+  type CreateTableStmt,
+  type AlterTableStmt,
+  type Program,
+} from 'sql-parser-cst';
+
+// Currently this script only cares about CREATE TABLE statements and ALTER
+// TABLE statements that add primary key constraints. All the other schema aspects of the
+// pg_dump are generally beyond the capability of SQLite. Perhaps index creation
+// can be added but it will get really tricky fast since SQLite's indices are
+// more more simplistic than postgres.
+
+const args = process.argv;
+const migrationsDir = resolve(join(__dirname, '..', 'migrations'));
+const sqliteSchemaDir = resolve(
+  join(__dirname, '..', '..', 'host', 'config', 'schema'),
+);
+const INDENT = '  ';
+
+let pgDumpFile = args[2];
+if (!pgDumpFile) {
+  console.error(`please specify the path of the pg_dump file`);
+  process.exit(-1);
+}
+let pgDump = readFileSync(pgDumpFile, 'utf8');
+
+let cst = parse(prepareDump(pgDump), {
+  dialect: 'postgresql',
+  includeSpaces: true,
+  includeNewlines: true,
+  includeComments: true,
+  includeRange: true,
+});
+
+let sql: string[] = [
+  `
+-- This is auto-generated from packages/realm-server/scripts/convert-to-sqlite.ts
```

Comment:
```suggestion
-- This is auto-generated by packages/realm-server/scripts/convert-to-sqlite.ts
```

It’s really created from the `pgDumpFile` right?

Reply:
Well yeah, but if you need to adjust how we transform the dump file you’ll want to adjust the converter tool.